src/user-agent/feature-support/is-webp-supported.ts

```diff
@@ -0,0 +1,23 @@
+import { NumericRange } from '../../mathf/numeric-range';
+import { Browser } from '../browser/base';
+import { Edge } from '../browser/edge';
+import { Firefox } from '../browser/firefox';
+import { Chrome } from '../browser/chrome';
+import { Opera } from '../browser/opera';
+import { isFeatureSupported } from './is-feature-supported';
+
+const webpSupportedBrowsers: Map<typeof Browser, NumericRange> = new Map<
+  typeof Browser,
+  NumericRange
+>([
+  [Edge, new NumericRange(18, Number.POSITIVE_INFINITY)],
+  [Firefox, new NumericRange(65, Number.POSITIVE_INFINITY)],
+  [Chrome, new NumericRange(32, Number.POSITIVE_INFINITY)],
+  [Opera, new NumericRange(19, Number.POSITIVE_INFINITY)]
+]);
+
+function isWebpSupported(browser: typeof Browser): boolean {
```

Comment:
Kind of a duplicate of is.supportigWebp() and is.supportingWebpAsync().

Although this version might be faster since just checks for browser version number.

Reply:
Removed user-agent/